Mixpanel/Mixpanel.swift

```diff
@@ -126,6 +132,9 @@ class MixpanelManager {
     }
 
     func removeInstance(name instanceName: String) {
+        if instances[instanceName] === mainInstance {
```

Comment:
This is a new thing that was added that allows users to get the Mixpanel instance anywhere in the code without keeping a reference to it when it was initialized